tests/helpers/test_entityfilter.py

```diff
@@ -44,6 +69,27 @@ def test_excludes_only_case_3():
     assert testfilter("sun.sun") is True
 
 
+def test_excludes_only_with_glob_case_3():
+    """If exclude specified, pass all but specified (Case 3)."""
+    incl_dom = {}
+    incl_glob = {}
+    incl_ent = {}
+    excl_dom = {"light", "sensor"}
+    excl_glob = {"cover.*_window"}
+    excl_ent = {"binary_sensor.working"}
+    testfilter = generate_filter(
+        incl_dom, incl_ent, excl_dom, excl_ent, incl_glob, excl_glob
+    )
+
+    assert testfilter("sensor.test") is False
+    assert testfilter("light.test") is False
+    assert testfilter("cover.bedroom_window") is False
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5745133)